docs/README.md

```diff
@@ -230,3 +233,434 @@ with vSphere. Below are the key integrations that are enabled by the vSphere clo
 ### Kubernetes Persistent Volumes
 
 ## Installing and Operating the vSphere Cloud Provider
```

Comment:
Nvm, figured it's easier to iterate on this PR. 

My biggest question is that this whole section feels more like a tutorial. Thoughts on moving this to `docs/tutorials/kubernetes-on-vsphere-with-kubeadm.md` and keeping `## Installing and Operating the vSphere Cloud Provider` a bit more generic?

Reply:
That would work, maybe it works best as a discrete guide and this section is more of a 1) set up VMs, 2) install required components, with links to relevant sections?

Reply:
Yeah, was thinking more like:
* `vsphere.conf` and `--cloud-config` points to it. 
* ensure `--cloud-provider=vsphere` is set
* etc 

And the tutorial would be a specific walk through of that